test/functional/core/job_spec.lua

```diff
@@ -274,10 +273,11 @@ describe('jobs', function()
   end)
 
   it('can omit options', function()
-    if helpers.pending_win32(pending) then return end
     neq(0, nvim('eval', 'delete(".Xtestjob")'))
-    nvim('command', "call jobstart(['touch', '.Xtestjob'])")
-    nvim('command', "sleep 100m")
+    local touch_cmd = iswin() and "'Out-File -encoding ASCII .Xtestjob'"
+                              or "['touch', '.Xtestjob']"
+    nvim('command', "call jobstart(" .. touch_cmd .. ")")
+    nvim('command', "sleep " .. (iswin() and "5" or "100m"))
```

Comment:
5s is expensive for such a trivial test. Can we just check that `jobstart()` returned `>0` instead? And instead of running "touch", run `nvim -u NONE ...`.

Reply:
`call jobstart(['nvim', '-u', 'NONE', '-c', 'q', '.Xtestjob'])`?
Are sleep and delete required if we're checking the job id only?

Reply:
I think not. IIUC the test only verifies that `jobstart()` works without the options dict. So the touch+sleep is pointless, and anything covered by it is well-trodden by other tests.